dataline-commons/src/test/java/io/dataline/commons/json/JsonsTest.java

```diff
@@ -102,9 +102,9 @@ void testToJsonNode() {
     Assertions.assertEquals(
         "{\"test\":\"abc\",\"test2\":\"def\"}",
         Jsons.jsonNode(
-                ImmutableMap.of(
-                    "test", "abc",
-                    "test2", "def"))
+            ImmutableMap.of(
```

Comment:
seems like we'll need to manually un-break lines which were broken at 80chars by the previous formatter, but not a big deal. I like that it caps us at 150 but doesn't force us to make full use of it in case it's more readable to break earlier.

Reply:
correct, I did it on purpose since automatic unwrapping looks really bad in some cases.